boa_tester/src/exec/mod.rs

```diff
@@ -57,53 +57,75 @@ impl TestSuite {
             println!();
         }
 
-        // Count passed tests
-        let mut passed = 0;
-        let mut ignored = 0;
-        let mut panic = 0;
+        // Count passed tests and es specs
+        let mut all = Statistics::default();
+        let mut es5 = Statistics::default();
+        let mut es6 = Statistics::default();
+
+        let mut append_stats = |spec_version: &SpecVersion, f: &dyn Fn(&mut Statistics)| {
```

Comment:
If `SpecVersion` is `Copy`, you can just use it without a reference, which makes comparisons simpler without an `&`.

Reply:
Ah, didn't know that. Thanks for the tip!